libgnucash/engine/Transaction.h

```diff
@@ -767,10 +768,10 @@ time64 xaccTransGetVoidTime(const Transaction *tr);
 void xaccTransDump (const Transaction *trans, const char *tag);
 #endif
 
-/** The xaccTransRecord() method iterates through the splits and
+/** The xaccTransRecordPrice() method iterates through the splits and
  *  and record the non-currency equivalent prices in the price database.
  */
```

Comment:
You need to document the parameters, e.g.
```
* @param trans The transaction whose price is recorded
* @param source The price priority level
```

Reply:
fixed